components/browse/HeroHeader.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="container hero-wrapper">
+    <b-jumbotron
+      :style="jumbotronStyle"
+      fluid
+      class="browse-jumbotron"
+      text-variant="white"
```

Comment:
This seems kind of contradictory, as the text in the banner is black/grey. This only affects the i icon for attribution. So perhaps remove this here, and add the white color there. Seems more clear.

Reply:
good point